  File "/Users/jorgepiedrafita/Documents/dev/ai_number_read/./numbers.py", line 5, in <module>
    import tensorflow_datasets as tfds
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/tensorflow_datasets/__init__.py", line 43, in <module>
    from tensorflow_datasets.core import tf_compat
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/tensorflow_datasets/core/__init__.py", line 22, in <module>
    tf_compat.ensure_tf_install()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/tensorflow_datasets/core/tf_compat.py", line 60, in ensure_tf_install
    tf_version = distutils.version.LooseVersion(tf.__version__)
AttributeError: module 'tensorflow' has no attribute '__version__'